Uxd 98 html table (#644)
acl-services/paprika: packages/Table/README.md

<!-- autogenerated don't modify -->

# @paprika/table - 0.1.0

## Description

> The Table component is a simple wrap for the html table, for more advance feature please take a look into the DataGrid

Reviewer: ```suggestion
The Table component is a simple wrap for the html table. For the use of a data table, please take a look into the DataGrid component.
```
